web/source/osk/oskManager.ts

```diff
@@ -1271,7 +1301,9 @@ namespace com.keyman.osk {
         if(device.touchable) {
           Ls.position='fixed';
           Ls.left=Ls.bottom='0px';
-          Ls.height=Ls.maxHeight=(<HTMLElement> this.vkbd.kbdDiv.firstChild).style.height;
+          let vkbdHeight = (<HTMLElement> this.vkbd.kbdDiv.firstChild).style.height;
+          vkbdHeight = vkbdHeight.substr(0, vkbdHeight.indexOf('px'));
+          Ls.height=Ls.maxHeight= (parseInt(vkbdHeight) + this.getBannerHeight()) + 'px';
```

Comment:
For [dumb reasons](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description), always include the radix when calling `parseInt`:

```suggestion
          Ls.height=Ls.maxHeight= (parseInt(vkbdHeight, 10) + this.getBannerHeight()) + 'px';
```